app/views/curation_concerns/file_sets/_show_actions.html.erb

```diff
@@ -9,11 +9,8 @@
       <%= link_to "Edit This #{@presenter.human_readable_type}", edit_polymorphic_path([main_app, @presenter]), class: 'btn btn-default' %>
       <%= link_to "Delete This #{@presenter.human_readable_type}", [main_app, @presenter], class: 'btn btn-danger', data: { confirm: "Delete this #{@presenter.human_readable_type}?" }, method: :delete %>
 
-      <%= button_tag t('sufia.single_use_link'),
-                  class: "copypaste btn btn-default",
-                  title: "Single-Use Link to File",
-                  data: { generate_single_use_link_url: curation_concerns.generate_show_single_use_link_url(@presenter.id) },
-                  id: "copy_link_#{@presenter.id}" %>
+      <%= link_to t('sufia.single_use_link'), curation_concerns.generate_download_single_use_link_path(@presenter),
```

Comment:
@awead `'sufia.single_use_link'` Should this be a curation concerns translation instead of a sufia translation?